src/server/server.js

```diff
@@ -0,0 +1,8 @@
+import chalk from 'chalk';
```

Comment:
Our `start` script is broken now

Reply:
And I think this is the entry point and should be named index, whereas what is currently index should be named app or server